HIPS/hip-1.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Do you just not like the name HIPS? 😉 We can change if you feel strongly, but I think we should standardize the convention as soon as possible and get people to consciously associate HIPs -> Hedera Improvement Proposals.

And I think the naming convention for files themselves, with them being sequentially ordered, is a huge benefit long term and for discoverability. That way, these naturally become the standards used as a convention, e.g., the ERC20, which came out of [EIP20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) has been incredibly successful and much more meaningful than only the date.

Reply:
Yeah, I don't like HIPS as it also means a body part. EIPS couldn't get confused with a body part. Plus the S in HIPS is not part of the acronym so it would be HIPs. Maybe sufficient to just drop the S and I'd be okay. It sounds cooler that way, more hip, one could say.

Reply:
Haha, well I'd agree there. So is your preference `proposals` or `HIP`? 

I don't feel strongly either way.. my overarching sentiment is simply that I don't think we should be afraid per se of forcing people to get familiar with seeing `HIP` or `HIPs`, as that's how I expect we'll use it in conversation 🤷‍♂ e.g. say it either `hip` like the body part, or, `h.i.p.`

Reply:
`HIP` works for me

Reply:
Renamed, but can always revisit if there's a consensus later 👍